common/service/dynamicconfig/constants.go

```diff
@@ -828,6 +831,12 @@ const (
 	WorkerThrottledLogRPS
 	// ScannerPersistenceMaxQPS is the maximum rate of persistence calls from worker.Scanner
 	ScannerPersistenceMaxQPS
+	// ScannerGetOrphanTasksPageSize is the maximum number of orphans to delete in one batch
+	ScannerGetOrphanTasksPageSize
+	// ScannerBatchSizeForCompleteTasksLessThanAckLevel is the number of tasks to query per call in the scavenger. The scavenger then uses the return of CompleteTasksLessThanAckLevel, to decide if a tasklist can be deleted. So it's better to keep it a relatively high number to let it be more efficient.
+	ScannerBatchSizeForCompleteTasksLessThanAckLevel
```

Comment:
Now it's being used for both completeTasksLessThanAckLevel and `getTaskForCheckExpiry`...
Do you think we should separate them? or rename it to something else that make more sense(and revise the comment too). 

I am open to either one.